package.json

```diff
@@ -1,35 +1,39 @@
 {
-  "name": "airbnb-style",
-  "version": "2.0.0",
+  "name": "eslint-config-dcos",
```

Comment:
Then it won't work with eslint. Eslint requires its plugins to have a specific naming conventions. For config packages it should start with `eslint-config-`